alfred-margaret.cabal

```diff
@@ -22,6 +22,8 @@ tested-with:
                    , GHC == 8.4.4
                      -- Stackage LTS 13.10.
                    , GHC == 8.6.3
+                     -- Stackage LTS 16.18.
+                   , GHC == 8.8.4
```

Comment:
Should we also remove the previous `tested-with` lines now? I don't think we tested this particular release with all of them, did we?

Reply:
Oh, good point. i actually did test it with LTS 13.10, but not any of previous ones.